sys-block/partclone/partclone-0.3.18.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic
+if [[ -z ${PV%%*9999} ]]; then
+	EGIT_REPO_URI="https://github.com/Thomas-Tsai/${PN}.git"
+	inherit git-r3
+else
+	[[ -n ${PV%%*_p*} ]] && MY_PV="${PV}"
+	SRC_URI="
+		https://github.com/Thomas-Tsai/${PN}/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz
+	"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="Partition cloning tool"
+HOMEPAGE="https://partclone.org"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="
+apfs btrfs +e2fs exfat f2fs fat fuse hfs jfs minix ncurses nilfs2 ntfs reiser4
+reiserfs static ufs vmfs xfs
+"
+
+RDEPEND="
+	dev-libs/openssl:*
+	e2fs? ( sys-fs/e2fsprogs )
+	btrfs? ( sys-apps/util-linux )
+	fuse? ( sys-fs/fuse:0 )
+	jfs? ( sys-fs/jfsutils )
+	ncurses? ( sys-libs/ncurses:0 )
```

Comment:
Similar to ncurses comment -- just do := if we're linking against it and it exposes a subslot.